### DISCUSSION TITLE
Some more additons?

### DISCUSSION THREAD

**Discussion by M0JXD** [forart/HyMPS#33] [General]
Hi,

I came across this repo while looking for the source for Traverso DAW. It's awesome, thank you!
To confirm, this is for open-source, cross platform media creation tools? I noticed some of those in the list are not cross platform but might be in the future, e.g. KWave.

With that in mind I think I've noticed some omissions:

**Audio/DAWs:**
Ardour - https://ardour.org/
LMMS - https://github.com/LMMS/lmms
Qtractor - https://www.qtractor.org/ - Unix only, however it uses Qt and JACK so it could be made to be cross platform.
Non DAW -  https://github.com/falkTX/non/ - Again, Unix only at the moment, but because it uses FLTK and JACK it could be ported to cross platform. Seems to be rebooted using CMAKE (promising!) here: https://github.com/Stazed

**Audio/Editors:**
Jokosher - https://github.com/mjumbewu/jokosher
Sweep -  https://github.com/kfish/sweep

**Audio/Collections:**
Most of these are Linux focused, but can be helpful for finding Open Source Cross Platform Apps.
https://linuxdaw.org/
https://linuxmusic.rocks/
https://wiki.linuxaudio.org/wiki/start
The Arch Wiki also has info regarding audio software: https://wiki.archlinux.org/title/List_of_applications/Multimedia#Audio
Powered by Faust has lots of cross-platform, open-source projects: https://faust.grame.fr/community/powered-by-faust/
Not strictly a resource but https://linuxmusicians.com/ can be helpful too.

**Video/Editors:**
Kdenlive - https://kdenlive.org/
Blender - https://www.blender.org/ (Has great video editing capabilities.)

**Video/Tools:**
OBS - https://obsproject.com/

If any/all of these are inappropriate please feel free to disregard. Thanks again!

**Comment by MarcoRavich** [comment]
Hi there, I'm the author of the collection.

Well, the HyMPS goal is a bit different from all the other collections, that's why I consciously neglected to list the more well-known/mature projects - like Ardour, Blender, etc. - in it.

Anyway Qtracktor is listed in [Sequencers page](https://github.com/FORARTfe/HyMPS/blob/main/Audio/Sequencers.md#--) and some falkTX stuff in other pages too.

About @Stazed: I need to check his repos, anyway feel free to add yourself.

Hope more contibution will come !

**Reply by M0JXD** [reply]
Hi,

Thanks for the clarification. Sorry I missed Qtractor before (I'm excited for the v1 release so that's probably how I overlooked it!)
FYI Non is not by falkTX (although he added LV2 support), the main developer took down his repo after falling out with others in Linux audio so it's only on falkTX's account for archival purposes.

I've remembered a few more, although they might still be too popular:

Audio/AI-Based:
Spleeter - https://github.com/deezer/spleeter
demucs is mentioned but could have it's own entry?

Audio/Tools:
fre:ac - https://github.com/enzo1982/freac/tree/v1.1.7

Video/Editors:
Openshot - https://github.com/OpenShot
Shotcut - https://github.com/mltframework/shotcut

I was also thinking instead of Audio/Synthesizers it could be Audio/Virtual Instruments? That way some drum projects like DrumGizmo, Geonkick, Hydrogen etc. could be added.

I only have basic markdown knowledge but I can maybe have a look at adding these and making some pull requests? Of course the list is your vision so I'd only add the ones you want.

**Reply by MarcoRavich** [reply]
> Sorry I missed Qtractor before (I'm excited for the v1 release so that's probably how I overlooked it!) FYI Non is not by falkTX (although he added LV2 support), the main developer took down his repo after falling out with others in Linux audio so it's only on falkTX's account for archival purposes.
> 
Will check. 
[Our organization](https://github.com/FORARTfe) started to archive open source adbandoned code too: 
https://github.com/orgs/FORARTfe/repositories

> I've remembered a few more, although they might still be too popular:
> 
> Audio/AI-Based: Spleeter - https://github.com/deezer/spleeter demucs is mentioned but could have it's own entry?
> 
Ouch, I've forgotten it !
Just added [here](https://github.com/FORARTfe/HyMPS/blob/main/Audio/AI-based.md#separating-).

> Audio/Tools: fre:ac - https://github.com/enzo1982/freac/tree/v1.1.7
> 
Already listed [here](https://github.com/FORARTfe/HyMPS/blob/main/Audio/DiscsTools.md#ripping-).

> Video/Editors: Openshot - https://github.com/OpenShot Shotcut - https://github.com/mltframework/shotcut
> 
I've listed the backend lib, but not the main repo.
Added.

> I was also thinking instead of Audio/Synthesizers it could be Audio/Virtual Instruments? That way some drum projects like DrumGizmo, Geonkick, Hydrogen etc. could be added.
> 
Good point, I'll consider it for the post-1.0 releses.

> I only have basic markdown knowledge but I can maybe have a look at adding these and making some pull requests? Of course the list is your vision so I'd only add the ones you want.
>
Well, just "grab" a pre-compiled tab row in your favourite flat-text editor ([Notepad++](https://notepad-plus-plus.org/) suggested) and modify it.
Example:
```|[Shotcut](https://github.com/mltframework/shotcut#readme)|A free, open source, cross-platform video editor|[![](https://img.shields.io/github/languages/top/mltframework/shotcut?color=pink&style=flat-square)](https://github.com/mltframework/shotcut/graphs/contributors)|[![](https://flat.badgen.net/github/license/mltframework/shotcut?label=)](https://github.com/mltframework/shotcut/blob/main/LICENSE)|[![](https://img.shields.io/github/last-commit/mltframework/shotcut?style=flat-square&label=)](https://github.com/mltframework/shotcut/graphs/code-frequency)|```

Then select ```mltframework/shotcut``` and CTRL+H (substitute) with another repo - short - reference.

That's all.
